.flake8

```diff
@@ -0,0 +1,3 @@
+[flake8]
+max-line-length = 95
+extend-ignore = E203, E712
```

Comment:
This should be in the pyproject.toml

Reply:
You and thousands of others want it, but the lead maintainer of Flake8 hates pyproject.toml, has locked the issue requesting it for years, and blocks anyone suggesting it be added.

When pip changes the default to PEP 517 builds (so adding a pyproject.toml file no longer changes the build type), this _might_ change (planned for 23.1, IIRC). Otherwise, the options are either using something besides flake8 (ruff is looking promising), or use a hack - but those tend to break every couple of patch releases because the flake maintainers intentionally change the internals they have to use to inject pyproject.toml support.

All other major tools that I'm aware of support pyproject.toml now, including mypy and tox, which were a bit slow in coming, but flake8 is still a strong holdout with no interest in changing.

Reply:
Oh, I didn't know that, sorry to hear.